pkg/controllers/multicluster/serviceexport_controller.go

```diff
@@ -225,6 +236,12 @@ func (r *ServiceExportReconciler) extractEndpoints(ctx context.Context, svc *v1.
 		return nil, err
 	}
 
+	clusterId := &aboutv1alpha1.ClusterProperty{}
```

Comment:
Since the parent Reconcile function already retrieved this, should we pass the value into the extractEndpoints function and save some unit test coverage percentage?

Reply:
sure!